app/styleguide/sg-components.tsx

```diff
@@ -0,0 +1,21 @@
+
+export function Example({children}) {
```

Comment:
I can, I didn't bother doing it because in my opinion the Styleguide itself, and it's related components are not mission critical. But I'll add the type annotations back in.

Reply:
Thank you @andrewebdev. I understand what you mean about the component not being mission critical, but I was thinking, since we decided on using type annotations, that we should just start applying it for consistency.